### PR TITLE
Milestone 1: Mobile First

### PR DESCRIPTION
**Pull request by CarlosIgreda**
## Portfolio Website Milestone 1: Mobile First

In this milestone I:
•	Created a new branch according to the GitHub flow rules
•	Added a .gitignore file
•	Set up the linters config
•	Used flexbox
•	Created hover effect
•	Created a UI
•	Used images and backgrounds to enhance the look of the website

### CODE REVIEW

**Review by ABDULALI3468 — COMMENTED** [review]
Hi @CarlosIgreda , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

> You have submitted a project, whose README file has a Microverse Logo on Top. Please consider either removing it or replacing it with your own logo only if you have one. Remember its a project requirement to have a standard and professional README.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by CarlosIgreda** [comment]
Hi @ABDULALI3468. I'm going to remove that Microverse Logo like you suggested to me. Thanks for the feedback and please let me know any other suggestions or advice.

**Comment by CarlosIgreda** [comment]
Hi again @ABDULALI3468. I updated the README.md file. Please could you give me some feedback about my project Thanks in advance.

**Comment by ABDULALI3468** [comment]
> Hi again @ABDULALI3468. I updated the README.md file. Please could you give me some feedback about my project Thanks in advance.

@CarlosIgreda Yes it's looking good now. You can go for the review. But I am seeing that you have closed your Pull Request consider reopening it and using the same PR for the review always for each milestone. In some cases, you can use another PR like the Code Reviewer asked you to it in some certain conditions.

**Comment by CarlosIgreda** [comment]
Ready @ABDULALI3468 . This PR is reopened.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+</head>
+
+<body>
+
+    <header id="topheader">
+        <nav id="topnav">
+            <div id="navdiv">
+                <a href="#" id="logoimg">
+                    <img id="photo" width: "50" height="50" src="/images/FotoPerfilCarlos.jpeg" alt="dev">
+                </a>
```
Comment (shella12) [inline index.html:27]: - [ ] [OPTIONAL] Photographs are not brandable. I would suggest not using your personal photo as a logo. A logo can be an icon, a symbol – perhaps a picture that represents something you sell or a value you stand for. Also, remember that your logo will likely need to be resized depending on where it's being placed; try to use something that looks clear and scalable.

> When photographs are used in logos, they negate the qualities that make for a good, brandable logo design. Photographs can make it more difficult for your audience to identify with your brand and run the risk of making your logo too generic.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @CarlosIgreda , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✅ 
- Professional README.md ✅   
- clean code ✅ 
- Good commit history ✅ 
- Descriptive pull request ✅ 
- Great use of semantic tags

## Required Changes ♻️

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (CarlosIgreda) [reply index.html]: Done @shella12  , I removed the photo.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+#idmain {
+  background-image: url(/images/header-shapes\ mobile@2x.svg);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
```
[the remark this answers is not in the document]
Comment (CarlosIgreda) [reply css/main.css]: Done @shella12, I added the "border-bottom-left-radius" like the figma template.

[the remark this answers is not in the document]
Comment (CarlosIgreda) [reply css/main.css]: The image that I was using was the mobile version, but I've just updated the content to have the same aspect

File: index.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                I'm Carlos <br> Glad to see you!
+            </h1>
+            <p id="idp1">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p><br>
+            <p id="idconnect">
+                LET'S CONNECT
+            </p>
```
[the remark this answers is not in the document]
Comment (CarlosIgreda) [reply index.html]: Done @shella12, I made the change and now is <a> tag.

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  font-family: 'Poppins', sans-serif;
+  color: #172b4d;
+}
+
+#idp1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #344563;
+}
```
[the remark this answers is not in the document]
Comment (CarlosIgreda) [reply css/main.css]: Done @shella12, I added the ```font-weight``` as you suggested to me.

File: css/main.css [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+#icons {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-items: left;
+  gap: 0.625rem;
+  margin: 0.938rem 0;
+  align-items: center;
+}
```
[the remark this answers is not in the document]
Comment (CarlosIgreda) [reply css/main.css]: Done @shella12, I made the update for ```gap```.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+            <a href="#">
+                <img id="hamburger" src="/images/Union.svg" alt="menu-icon">
+            </a>
+        </nav>
+    </header>
+    <main id="idmain">
+        <section id="idsection">
+            <h1 id="idh1">
+                I'm Carlos <br> Glad to see you!
+            </h1>
```
[the remark this answers is not in the document]
Comment (CarlosIgreda) [reply index.html]: Done @shella12, I put Mariane instead of my name (like the figma template). Also, I updated the ```font-size```.

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @CarlosIgreda  👋

Great work implementing the changes suggested by the reviewer. You nailed the project 💪 Keep up the great work 🥇

**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Commits are descriptive, good work.
-  ✅ Linters are working properly, great job! 
- ✅ Well done implementing the optional suggestion

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CarlosIgreda** [comment]
🥳🥳🥳 Thanks for the feedback!

**Pull request merged**